### ISSUE TITLE
Word Wrap

### ISSUE THREAD

**Issue by benthayer** [benthayer/git-gud#274]
Words tend to be cut off and finished on a new line when we just print them out directly, so word wrap would be a great usability feature.

**Comment by sahansk2** [comment]
A combination of the [`textwrap`](https://docs.python.org/3.6/library/textwrap.html) module and [`shutil.get_terminal_size()`](https://docs.python.org/3/library/shutil.html#querying-the-size-of-the-output-terminal) should do the trick. Using `from builtins import print as python_print` may be useful if we want to define our own `print` that wraps the builtin printing function. 

This shouldn't be *too* difficult to implement, so I won't take this issue in case there's anyone who'd like to work on the issue. I'd guess that the code for this new print function should be in `gitgud/skills/user_messages.py`.

**Comment by benthayer** [comment]
That's one option. We can actually modify the print function output stream directly, but we'd need that to be the first thing executed in the code so that the change is in effect. This would reduce refactoring and it'll make sure we're always using the right function. I'm not sure if I'm a fan of this method, but it's possible. I do like the idea of having all output messages in user_messages.py though, even more than are there now, which would eventually help with adding multiple languages

**Comment by benthayer** [comment]
I agree this would be a good issue for newcomers. I added the appropriate tags. Anyone interested, feel free to comment
